graphql/http_handler.go

```diff
@@ -57,12 +60,14 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 			queryPath = fieldCtx.Path().String()
 			args = fieldCtx.Args
 		}
-		grip.Error(message.WrapError(err, message.Fields{
-			"path":    "/graphql/query",
-			"query":   queryPath,
-			"args":    args,
-			"request": gimlet.GetRequestID(ctx),
-		}))
+		if err != nil && !strings.HasSuffix(err.Error(), ContextCanceledErrorMessage) {
```

Comment:
Sorry for the confusion I meant something like this.
```suggestion
		if !errors.Is(err, context.Canceled) {
```

Reply:
`errors.Is` just looks for the error in the error tree so if it appears anywhere this condition should be met.